src/core/filter/security.go

```diff
@@ -147,6 +153,50 @@ func (s *secretReqCtxModifier) Modify(ctx *beegoctx.Context) bool {
 	return true
 }
 
+type robotAuthReqCtxModifier struct{}
+
+func (r *robotAuthReqCtxModifier) Modify(ctx *beegoctx.Context) bool {
+	robotName, robotTk, ok := ctx.Request.BasicAuth()
+	if !ok {
+		return false
+	}
+	log.Debug("got robot information via token auth")
```

Comment:
This is misleading, this information will be printed even when the request contains basic auth info for a regular user.